_extras/guide.md

```diff
@@ -5,16 +5,20 @@ title: "Instructor Notes"
+
+**Timing** Unfortunately these episodes are incredibly cumulative and there is not much that can be cut along the way. 
+If you are running short on time, we recommend cutting or stream-lining parts of episode 7 about plotting. It is also possible to stream-line some of the CSV section at the end of episode 5, for instance skip the read back section which demonstrates how extra columns can slip into CSV files.
 
-Because this lesson follows a single dataset throughout, its easy for learners (and instructors) to lose sight of the bigger picture and focus instead on the scientific goals. At the beginning of each lesson it is recommended that the instructor discuss both the scientific goal of the lesson (with frequent references to Figure 1) and highlight the big picture skills that we hope each student takes away from the lesson, beyond the specific science case. At the end of the lesson the instructors should recap the same information, highlighting the best practices covered. TODO: reference slide show.
 ### Decisions Made
-* We explicitly recommend using Jupyter notebooks rather than Jupyter lab as we've found them more reliable across different platforms
-### Introduction
+* We explicitly recommend using Jupyter notebooks rather than Jupyter lab as we have found them more reliable across different platforms and they require less up front explanation.
 
-* Astronomical datasets are getting larger and larger. As a result, the days of downloading a raw image and performing the end to end analysis are numbered. Instead, already reduced data (and simulations) are being stored in databases. If you know how to ask these databases the right questions, they are an incredibly powerful tool. The ability to interact with these large databases is fundamental to the future of astronomy as well as a skill that is highly valued outside of astronomy.
+### Introduction
+* Astronomical datasets are getting larger and larger. As a result, the days of downloading a raw image and performing the end to end analysis are numbered. Instead, already reduced data (and simulations) are being stored in databases. If you know how to ask these databases the right questions, they are an incredibly powerful tool. The ability to interact with these large databases is fundamental to the future of astronomy as well as a skill that is highly valued outside of the astronomical community.
```

Comment:
I think we can/should remove "As a result, the days of downloading a raw image and performing the end to end analysis are numbered." since it is unlikely that this will happen within the next 10-15 years. I think we could simply say, "As a result, high-level science products are often stored in databases."

Reply:
What about something like: As a result, astronomical analysis is shifting away from working with raw imaging and towards analyzing already reduced data (and simulations) that are stored in large databases.